src/main/charts/jira/templates/statefulset.yaml

```diff
@@ -42,6 +42,13 @@ spec:
               {{ end }}
             {{- include "jira.databaseEnvVars" . | nindent 12 }}
             {{- include "jira.clusteringEnvVars" . | nindent 12 }}
+            {{ with .Values.jira.license.secretName }}
+            - name: JIRA_SETUP_LICENSE
+              valueFrom:
+                secretKeyRef:
+                  name: { { . } }
+                  key: { { $.Values.jira.license.secretKey } }
```

Comment:
weird spacing of the `{{` and `}}` here, could you make it consistent with the rest of them?

Reply:
Good catch, must have had auto-formatting during commit enabled.